src/components/DisplayComponent.js

```diff
@@ -14,6 +24,33 @@ class DisplayComponent extends Component {
   }
 
   render() {
+    var display;
+    if (this.state.displayComponent === 'Chart') {
+      display = (
+        <BarChart
+          width={600}
+          height={300}
+          data={[
+            { name: 'Page A', uv: 4000, pv: 2400, amt: 2400 },
+            { name: 'Page B', uv: 3000, pv: 1398, amt: 2210 },
+            { name: 'Page C', uv: 2000, pv: 9800, amt: 2290 },
+            { name: 'Page D', uv: 2780, pv: 3908, amt: 2000 },
+            { name: 'Page E', uv: 1890, pv: 4800, amt: 2181 },
+            { name: 'Page F', uv: 2390, pv: 3800, amt: 2500 },
+            { name: 'Page G', uv: 3490, pv: 4300, amt: 2100 }
+          ]}
+          margin={{ top: 5, right: 30, left: 20, bottom: 5 }}
```

Comment:
I see you do use inline css. That case, we can use both `scss` and `jss` lib in parallel

Reply:
This is not inline css. This is a prop to the Barchart component in recharts. If the props margin is set then the component is setting the margin css. I think this is a cleaner way to set css. I you feel we should move it elsewhere do let me know! I'd be happy to do that.